core/src/main/java/com/amplitude/core/Amplitude.kt

```diff
@@ -43,12 +48,12 @@ import java.util.concurrent.Executors
  */
 open class Amplitude(
     val configuration: Configuration,
-    val store: State,
```

Comment:
I'm not sure we can do this at this point as this will break any sub-class or usage of core Amplitude

Reply:
As discussed, these is public/open, but since Unified SDK is a breaking change then we're good with it